app/classes/Langchecker/Project.php

```diff
@@ -28,36 +28,26 @@ public static function getReferenceLocale($website)
     }
 
     /**
-     * Return supported files for website
+     * Return supported locales for a file in website. If not defined,
```

Comment:
Uhm, I actually meant to use "website" here, since we call them websites internally. Would project be less confusing?